core/src/main/res/values-v31/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="widget_inner_radius">@android:dimen/system_app_widget_inner_radius</dimen>
```

Comment:
To be honest, I'm not really convinced about the system radius. It makes the cover almost round on my device, which we never do inside the app. I would rather use a fixed radius like in the rest of the app.

Reply:
Looking at it again it's probably better to use 4dp on all Android versions.